contracts/src/payments/SuccinctFeeVault.sol

```diff
@@ -94,7 +86,7 @@ contract SuccinctFeeVault is IFeeVault, TimelockedUpgradeable {
             revert InsufficentAllowance(_token, _amount);
         }
 
-        token.transferFrom(msg.sender, address(this), _amount);
+        token.safeTransferFrom(msg.sender, address(this), _amount);
```

Comment:
This is a recommendation from an old audit report - though not urgent since the problem only effects tokens that don't revert on a failed transfer